### DISCUSSION TITLE
Update progress bar description

### DISCUSSION THREAD

**Discussion by dariush-bahrami** [Textualize/rich#1347] [Q&A] [answered]
**Hi, Is there any way to update the progress bar description?** 
I want to train a neural network with PyTorch. Using rich for training loops is very pleasant but the problem is static progress bar description does not help so much when training a neural network. I want to print each epoch loss in the description to get an idea of training progress.

**Comment by willmcgugan** [comment, marked as answer]
You can update the description with the update method

https://rich.readthedocs.io/en/latest/reference/progress.html#rich.progress.Progress.update

**Reply by dariush-bahrami** [reply]
Thanks for your response. I use `rich.progress.track`; Is there anyway to update the track description? If no, where can I find an example code to use `rich.progress.Progress`?

**Reply by willmcgugan** [reply]
https://rich.readthedocs.io/en/latest/progress.html#advanced-usage
